### ISSUE TITLE
allow rendering comparison chart split by top level bgroup

### ISSUE THREAD

**Issue by cartazio** [bos/criterion#62]
[heres an example](http://www.wellposed.com.s3.amazonaws.com/examples-cached/rendering-woes/summary-multi.html) of splitting the top level chart vs [the current one](http://www.wellposed.com.s3.amazonaws.com/examples-cached/rendering-woes/summary-works.html)

https://bitbucket.org/carter/multicriterion-templates/src/b1c1b808b71c?at=master is the current repo for this.

I'm not necessarily saying "merge this in", but might at least be worth having a curated "plugins.md" / "addons.md" / "moretools.md" list or something that is centrally curated

**Comment by rrnewton** [comment]
I'd also like this -- @cartazio could you turn it into a PR?

**Comment by cartazio** [comment]
I've been meaning to de bit rot this for a while, and make it more
performant at rendering. Currently it has some gnarly / huge quadratic
blowups ok large data sets / bench suite rendering

 I'll be happy to turn it into a PR if you mean that you're the primary
maintainer now and can Shepard it being merged in, or something like that.

**Comment by rrnewton** [comment]
Yes, exactly, I'm a co-maintainer and we're trying to catch up on outstanding issues.

**Comment by cartazio** [comment]
Sweet.  Would an updated template in the style of this ticket but saner
rendering perf be a candidate for being the new default if it's
intelligible ?

**Comment by cartazio** [comment]
@rrnewton this is on my todo queue now. @EleanorWhite and I are going to see about maybe finishing it up :)
